src/lib/containers/TermsAndConditions.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useState } from 'react'
+import { tcItem } from './TermsAndConditionsItem'
+import TermsAndConditionsItem from './TermsAndConditionsItem'
+import { ReactComponent as chatBubblesSvg } from '../assets/icons/terms/chat-bubbles.svg'
+import { ReactComponent as lockSvg } from '../assets/icons/terms/lock.svg'
+import { ReactComponent as scaleSvg } from '../assets/icons/terms/scale.svg'
+import { ReactComponent as shieldSvg } from '../assets/icons/terms/shield.svg'
+import { ReactComponent as peopleSvg } from '../assets/icons/terms/people.svg'
+import { ReactComponent as penSvg } from '../assets/icons/terms/pen.svg'
+import { ReactComponent as speakerSvg } from '../assets/icons/terms/speaker.svg'
+import { ReactComponent as flagSvg } from '../assets/icons/terms/flag.svg'
+import { ReactComponent as RefreshSvg } from '../assets/icons/terms/refresh.svg'
+import { Button, Modal } from 'react-bootstrap'
+import SignatureCanvas from 'react-signature-canvas'
+import ReactSignatureCanvas from 'react-signature-canvas'
+
+export type TermsAndConditionsProps = {
+  token?: string
+}
+
+const tcList: tcItem[] = [
+  {
+    icon: chatBubblesSvg,
+    label: 'I will adhere to the Synapse Community Standards of inclusion and respect.',
+    description: `A participant in the Synapse Community: <ul>
+    <li>Welcomes others</li>
+    <li>Uses inclusive language</li>
+    <li>Shares experiences and knowledge</li>
+    <li>Respects other viewpoints and ideas</li>
+    <li>Shows empathy and kindness when interacting with others</li>
+    </ul>
+Please refer to our full Synapse Community Standards.
+`,
+  },
+  {
+    icon: lockSvg,
+    label: 'I will adhere to all conditions and use limitations, including privacy laws and regulations.',
+    description: ''
+  },
+  {
+    icon: scaleSvg,
+    label: 'I will act ethically and responsibly.',
+    description: ''
+  },
+  {
+    icon: shieldSvg,
+    label: 'I will use appropriate physical, technical and administrative measures to keep data secure and protect participants’ privacy.',
+    description: ''
+  },
+  {
+    icon: peopleSvg,
+    label: 'I will support open access best practices for public facing data.',
+    description: ''
+  },
+  {
+    icon: penSvg,
+    label: 'I will credit research participants and all data sources.',
+    description: ''
+  },
+  {
+    icon: speakerSvg,
+    label: 'I will not use data for marketing and/or advertising.',
+    description: ''
+  },
+  {
+    icon: flagSvg,
+    label: `I will report any suspected data breach or misuse to <a href="mailto:privacyofficer@sagebase.org">privacyofficer@sagebase.org</a> within 2 business days of initial incident knowledge.`,
+    description: ''
+  },
+]
```

Comment:
Not sure if the terms and conditions should live somewhere outside the component. If so, where would be a good place to store this?

Reply:
Suspect that this informed consent text should not change often, but you bring up a good point.  One possibility (that we've used elsewhere in this project) is to use a Synapse Table for the structured content.  Or it could be an external json file.  In this particular case, I don't think we need to externalize this text/copy.